codebase/src/cpp/syscommon/src/InputBuffer.cpp

```diff
@@ -41,23 +41,23 @@ InputBuffer::~InputBuffer()
 //----------------------------------------------------------
 //                    INSTANCE METHODS
 //----------------------------------------------------------
-unsigned char InputBuffer::readUInt8() throw ( IOException )
+unsigned char InputBuffer::readUInt8()
```

Comment:
From memory the signature of the method definition also needs to include the noexcept qualifier to match the declaration signature otherwise GCC will complain (MSVC is a bit more relaxed about it).

Reply:
Have just checked, and only `noexcept` must be present on both function declaration and impl.

As `noexcept( false )` is the same as having no exception specification.

https://godbolt.org/z/zMrGPsvoG